packages/discovery-provider/src/queries/get_track_signature.py

```diff
@@ -87,7 +86,7 @@ def get_track_stream_signature(args: GetTrackStreamSignature):
     authed_user = get_authed_or_managed_user(user_data, user_signature, user_id)
 
     # all track previews and non-stream-gated tracks should be publicly available
-    if is_preview or not is_stream_gated:
+    if is_preview:
```

Comment:
i think we still need to check whether it's stream gated?

Reply:
got it